lib/index.js

```diff
@@ -1,95 +1,50 @@
 'use strict';
 
-const fs = require('fs');
-const path = require('path');
-const bemNaming = require('bem-naming');
 const Readable = require('stream').Readable;
-const schemes = require('./schemes');
-
+const async = require('async');
+
+const walkers = require('./walkers');
+
+/**
+ * Scans levels in file system.
+ *
+ * If file or directory is valid BEM entity then will be called `add` with info about this file.
```

Comment:
If file or directory is valid BEM entity then `add` will be called with info about this file.